map_data/geographical_region.txt

```diff
@@ -794,25 +822,104 @@ custom_eastern_kingdoms_jungles = {
 }
 custom_jungles_except_pandaria = {
 	regions = {
-		world_northrend_sholazar_basin
 		custom_eastern_kingdoms_jungles
 		custom_south_seas_except_deserts
+		world_northrend_sholazar_basin
 	}
 }
 custom_jungles = {
 	regions = {
 		custom_jungles_except_pandaria
+		world_kalimdor_feralas
 		world_pandaria_jade_forest
 		world_pandaria_krasarang_wilds
 	}
+	duchies = {
+		d_echo_isles
+		d_seething_shore
+		d_glaive_isle
+       		d_lost_isles
+	}
+}
+custom_forest = {
+	regions = {
+		world_eastern_kingdoms_quelthalas_quelthalas
+		world_eastern_kingdoms_quelthalas_zulaman
+		world_kalimdor_ashenvale
+		world_kalimdor_hyjal
+	}
+	duchies = {
+		
+		#Azshara#
+		d_haldarr
+		d_blackmaw
+		d_jozaes
+		d_legash
```

Comment:
Why not just refer to the Azshara region.

Reply:
d_elderath isnt forest, but I assume you would rather I just add it?

Reply:
> d_elderath isn't forest, but I assume you would rather I just add it?

Yes, it's in a forest-dominated region anyway.